packages/server/src/validators/UsersValidator.js

```diff
@@ -0,0 +1,27 @@
+import Models from '../models'
+import BaseValidator from './BaseValidator'
+import { isAddress, toChecksumAddress } from 'web3-utils'
+
+const { UserAddress } = Models
+
+class UsersValidator extends BaseValidator {
+  async validateForCreate({ email, address }) {
+    this._validateEmail(email)
+    await this._validateAddress(address)
+    return this.resetErrors()
+  }
+
+  _validateEmail(email) {
+    if (!email) this.addError({ email: 'An email address must be given' })
+  }
+
+  async _validateAddress(address) {
+    if (!isAddress(address)) return this.addError({ address: 'Given address is not valid' })
```

Comment:
Does `isAddress()` check for the checksum?

Reply:
If the address provided is checksumed, it does

Reply:
As discussed afterwards, it would most likely be best to always validate the checksum since this user generated information.